articles/purview/tutorial-using-python-sdk.md

```diff
@@ -0,0 +1,335 @@
+
+In this tutorial, you will learn how to:
+
+*   Grant the required rights to work programmatically with Purview
+*   Register a Blob Storage container as a data source in Purview
+*   Define and run a scan
+*   Search the catalog
+*   Delete a data source
+
+## Prerequisites
+
+For this tutorial, you will need :
+*   Python 3.6+
+*   An active Azure Subscription 
+*   An Azure Active Directory tenant associated with you subscription
+*   An Azure storage account
+*   A Purview account
+
+
+## Give Purview access to the Storage account
+
+Before being able to scan the content of the Storage account, you need to give purview the right role. 
+Go to your Storage Account via the portal. Select Access Control (IAM). Click on the Add button and select “Add role assignment”.
+
+:::image type="content" source="media/tutorial-using-python-sdk/add-role-assignment-storage.png" alt-text="Screenshot that shows Access .Control menu of the Storage Account"::: 
+
+In the next window, search for the “Storage blob Reader” role and select it:
+
+:::image type="content" source="media/tutorial-using-python-sdk/storage-blob-reader-role.png" alt-text="Screenshot that shows the Storage Blob Reader role in the list of available roles"::: 
+
+Then go on the “Members” tab and click on “Select members”:
+
+:::image type="content" source="media/tutorial-using-python-sdk/select-members-blob-reader-role.png" alt-text="Screenshot that shows where to click to grant the role the Purview Managed System Identity"::: 
+
+A new pane appears on the right. Search and select the name of your existing purview instance.
+You can then click on "Review + Assign". Purview now has the required reading right to scan your Blob Storage. 
+
+
+## Grant your application the access to your Purview account
+1. First, you need a [service principal with a secret](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#register-an-application-with-azure-ad-and-create-a-service-principal).
+Get the following information from your service principal and secret, they will be needed later:
+    *    Client ID (or Application ID)
+    *    Tenant ID (or Directory ID)
+    *    Client secret 
+    
+2. You now need to give the relevant Purview roles to your service principal. To do so, access the Purview instance. Select "Open Purview Studio" or open [Azure Purview's home page](https://web.purview.azure.com/) and choose the instance that you deployed.
+
+    Inside Purview Studio, go to the collections:
+    :::image type="content" source="media/tutorial-using-python-sdk/purview-collections.png" alt-text="Screenshot that shows how to access Purview collections"::: 
+
+    Select the collection you want to work with, and go on the "Role assignments" tab. Add the service principal (using its name) in the following roles: Collection admins, Data source admins, Data curators, Data readers.
```

Comment:
Please also provide a screenshot of at least one example of adding the service principal to one of these roles.